### PR TITLE
Add snowflake python connector

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
Hi @smtakeda -- Do you or any of your team want to be listed as maintainers of this recipe on conda-forge?

File: recipes/snowflake/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ef4cdac45402561dd2bf16e217cd7915b9fc2fd2444ed6c2715b8a6e90a3afbf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+    build:
```
Comment (marcelotrevisani) [inline recipes/snowflake/meta.yaml]: please, change that to ``host`` and add ``pip`` as a dependency

Comment (ericdill) [reply recipes/snowflake/meta.yaml]: Thanks @marcelotrevisani ... I'm still a bit wrapped around the axle with the conda-build 2 vs conda-build 3 syntax here 😸

**Comment by ericdill** [comment]
This is blocked by https://github.com/conda-forge/azure-storage-feedstock/pull/7

**Comment by ericdill** [comment]
Argh... Is `namespace_packages` still an issue with conda-build? I see a link from @msarahan [here](https://github.com/conda-forge/staged-recipes/wiki/Namespace-packages)

And I now see that the snowflake-connector-python package is kind of a mess with its folder structure and its [setup.py](https://github.com/snowflakedb/snowflake-connector-python/blob/master/setup.py)

**Comment by xhochy** [comment]
Exists already https://github.com/conda-forge/snowflake-connector-python-feedstock nowadays.

**Pull request closed without merging**